suite-native/module-settings/src/components/AboutViewOnlyBottomSheet.tsx

```diff
@@ -0,0 +1,77 @@
+import {
+    BottomSheet,
+    Box,
+    Button,
+    TitleHeader,
+    VStack,
+    Text,
+    OrderedListItem,
+} from '@suite-native/atoms';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { Translation } from '@suite-native/intl';
+
+const buttonWrapperStyle = prepareNativeStyle(utils => ({
+    gap: utils.spacings.medium,
+}));
+
+type AboutViewOnlyBottomSheetProps = {
+    isVisible: boolean;
+    onClose: () => void;
+};
+
+export const AboutViewOnlyBottomSheet = ({ isVisible, onClose }: AboutViewOnlyBottomSheetProps) => {
+    const { applyStyle } = useNativeStyles();
+
+    let order = 1;
+
+    if (!isVisible) {
+        return null;
+    }
+
+    return (
+        <BottomSheet isVisible={isVisible} onClose={onClose} isCloseDisplayed={false}>
+            <VStack spacing="large" paddingHorizontal="small">
+                <TitleHeader
+                    title={<Translation id="moduleSettings.viewOnly.about.title" />}
+                    subtitle={<Translation id="moduleSettings.viewOnly.about.subtitle" />}
+                />
+                <VStack>
+                    <Text variant="callout">
+                        <Translation id="moduleSettings.viewOnly.about.contentTitle" />
+                    </Text>
+                    <VStack spacing={0}>
+                        <Translation
+                            id="moduleSettings.viewOnly.about.content"
+                            values={{
+                                li: chunk => {
+                                    const index = order;
+                                    order += 1;
```

Comment:
Is it worth using variable `order` here? I believe it can be enough to put the order numbers directly inside translation string here same as we do in other modals. I would prefer to delete OrderedListItem component completely.

Reply:
I prefer in this kind of components, if they are defined by data and we dont have to change the code, when the texts change. Lets talk this on standup :)

Reply:
I had to keep the order out of the string to enable left padding in case of text breaking into multiple lines. To make it work with various text sizes I removed the fixed width of order.

b410c0a136

<img width="273" alt="Screenshot 2024-04-24 at 13 09 27" src="https://github.com/trezor/trezor-suite/assets/2011829/c45f8eef-da20-480d-8184-8ae012a68f07">